JSDemos/Demos/DataGrid/BatchUpdateRequest/description.md

```diff
@@ -0,0 +1,15 @@
+If **editing**.[mode](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/#mode) is *"batch"*, end-users can modify multiple records and submit all changes simultaneously. If data is stored on a server, the **DataGrid** component sends multiple requests to save edited objects - one request per object. This is because most servers process only one edit operation at a time. However, if your server supports batch update, you can configure the **DataGrid** to save all changes with a single request.
```

Comment:
```suggestion
*"Batch"* **edit**.[mode](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/#mode) enables users to modify multiple records and submit all changes simultaneously. If data is stored on a server, the **DataGrid** component sends multiple requests to save edited objects - one request per object. This is because most servers process only one edit operation at a time. However, if your server supports batch update, you can configure the **DataGrid** to save all changes in a single request.
```

Reply:
or 
```suggestion
If **editing**.[mode](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/#mode) is *"batch"*, users can modify multiple records and submit all changes simultaneously. If data is stored on a server, the **DataGrid** component sends multiple requests to save edited objects - one request per object. This is because most servers process only one edit operation at a time. However, if your server supports batch update, you can configure the **DataGrid** to save all changes with a single request.
```